draft-ietf-taps-interface.md

```diff
@@ -434,7 +434,7 @@ preference levels:
    |------------|------------------------------------------------------------------------|
    | Require    | Select only protocols/paths providing the property, fail otherwise     |
    | Prefer     | Prefer protocols/paths providing the property, proceed otherwise       |
-   | Ignore     | Cancel any system default preference for this property                 |
+   | Neutral    | No preference                                                          |
```

Comment:
I agree that `neutral` is better (and less overloaded) than `ignore`, but I'm not 100% sold on it.

I'm also wondering if it is useful to have an explicit way in the API to request to get the `default` system behavior for a property. I know that defaults are specified for individual properties below, but it may be useful for an application being ported between different implementation to say "just do what the system/implementation wants for this one", which is effectively neutral and passes the decision down to the lower layer.

If we replaced `neutral` with `default`, of course, it would be silly to say that the default disposition is `default`... but it's something to think about.

For various APIs we've exposed in the past, we've later had to add an explicit way to set the default value in this way.

Reply:
But now we have two voices in favor of "Ignore" again - the latest agreement was that it should be "Ignore", but without the pesky text "Cancel any system default preference for this property".  Combining this idea with what you say here, I suggest to call this:

> Ignore   -   No preference; use system defaults for this property

(just the opposite of "Cancel system defaults", actually  :)  )